spec/tests_unit/rubocop_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require_relative '../helpers/spec_helper.rb'
 
-JSON_FORMAT_COMMAND = 'rubocop  --except Metrics -f j 2>&1'
+JSON_FORMAT_COMMAND = 'rubocop  -f j --except Metrics 2>&1'
```

Comment:
I'm curious what these flags do?

Reply:
These flags try to control the output of rubocop, and these two flags would have the same results, so to make the test pass, I changed the code from the old one to the new one.